utils/util.go

```diff
@@ -102,3 +102,13 @@ func TrimSplit(str, sep string, count int) []string {
 
 	return result
 }
+
+func IsRegionService(name string) bool {
+	if strings.Contains(name, "/") {
```

Comment:
there is no real need for this if condition here. Split will generate an array of 1 if that happens